tests/diffusers_tests/test_t5_film_transformer.py

```diff
@@ -4,7 +4,7 @@
 
 import mindspore as ms
 
-from .modeling_test_utils import compute_diffs, generalized_parse_args, get_modules
+from ..modeling_test_utils import compute_diffs, generalized_parse_args, get_modules
 
 THRESHOLD_FP16 = 1e-2
```

Comment:
why 1e-2

Reply:
This is the unified baseline under diffuser tests.